platforms/documentation/docs/src/snippets/java-platform/dep-man/groovy/app/build.gradle

```
@@ -0,0 +1,7 @@
+plugins {
+    id 'java-library'
+}
+
+dependencies {
+    implementation platform(':platform')
+}
```

```suggestion
plugins {
    id("java-library")
}

dependencies {
    implementation(platform(":platform"))
}
```

Let's use consistent notation everywhere for applying plugins and declaring dependencies, regardless of language